apps/gdevelop/gdevelop.yml

```diff
@@ -0,0 +1,25 @@
+name: 'GDevelop'
+description: 'Create games without programming.'
+website: 'https://gdevelop-app.com/'
+repository: 'https://github.com/4ian/GDevelop'
+keywords:
+  - 'Game Editor'
+  - 'Open Source'
+  - 'Cross-Platform '
+  - 'Visual Programming'
+  - 'Online Editor'
+  - 'Piskel Sprite Editor'
+  - 'Google AdMob'
+license: MIT
+category: 'Graphics & Design'
```

Comment:
While accessible to designers, it's mostly a way to build games, i.e: develop. So I think it should be 'Developer Tools'

Reply:
That can be changed surely :)